src/app/products/home/home.component.spec.ts

```diff
@@ -8,10 +8,9 @@ describe('HomeComponent', () => {
 
   beforeEach(async () => {
```

Comment:
No test case here as well. 
We have test cases for presentational components and test cases for smart components.
You cannot ignore the presentational components simply because it is just UI. That is what makes them presentational hence they need to be tested